app.js

```diff
@@ -0,0 +1,268 @@
+'use strict';
+
+const addTaskForm = document.querySelector('#addTaskForm');
+const currentTasks = document.querySelector('#currentTasks');
+const completedTasks = document.querySelector('#completedTasks');
+const incr = document.querySelector('#incr');
+const decr = document.querySelector('#decr');
+const toDoHeading = document.querySelector('#toDoHeading');
+const completedHeading = document.querySelector('#completedHeading');
+const tasksContainer = document.querySelector('#tasksContainer');
+const clearAll = document.querySelector('#clearAll');
+const editTaskForm = document.querySelector('#editTaskForm');
+let currentEditId = '';
+
+clearForRender();
+renderTasks();
+
+if (!localStorage.getItem(0)) {
+	localStorage.setItem(0, 'incr');
+}
+
+incr.addEventListener('click', () => {
+	localStorage.setItem(0, 'incr');
+	clearForRender();
+	renderTasks();
+});
+
+decr.addEventListener('click', () => {
+	localStorage.setItem(0, 'decr');
+	clearForRender();
+	renderTasks();
+});
+
+clearAll.addEventListener('click', () => {
+	localStorage.clear();
+	localStorage.setItem(0, 'incr');
+	clearForRender();
+	renderTasks();
+});
+
+addTaskForm.addEventListener('submit', e => {
+	e.preventDefault();
+	const task = getNewTaskData();
+	addTaskToLocalStorage(task);
+	clearForRender();
+	renderTasks();
+	addTaskForm.reset();
+	addTaskForm.querySelector('[data-dismiss="modal"]').click();
+});
+
+tasksContainer.addEventListener('click', e => {
+	if (e.target.classList.contains('btn-danger')) {
+		const id = e.target.closest('.list-group-item').dataset.id;
+		deleteTask(id);
+		clearForRender();
+		renderTasks();
+	}
+
+	if (e.target.classList.contains('btn-success')) {
+		const id = e.target.closest('.list-group-item').dataset.id;
+		toggleCompleteness(id);
+		clearForRender();
+		renderTasks();
```

Comment:
it can be a reusable method

Reply:
Thanks. I refactored it.
```suggestion
		render();
```